core/common/src/main/java/alluxio/AlluxioEvent.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import alluxio.collections.Pair;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * Defines and Manages well-known Alluxio events.
+ *
+ * TODO(ggezer): Add per-process log4j configuration before adding events to other processes.
+ */
+public enum AlluxioEvent {
+  // Alluxio events.
+  // Next id:1019.
+  MasterProcessCreated(1000),
+  JournalSystemStarted(1001),
+  JournalSystemStopped(1002),
+  JournalSystemGainedPrimacy(1003),
+  JournalSystemLostPrimacy(1004),
+  MasterRpcServerStarted(1005),
```

Comment:
Are these related to `MasterProcessCreated` event? Should they be grouped together?

Reply:
grouped how?